docs/02-app/01-building-your-application/01-routing/03-linking-and-navigating.mdx

```diff
@@ -117,48 +124,55 @@ export default function Page() {
+There are two ways routes are prefetched in Next.js:
 
-The new router has an **in-memory client-side cache** that stores the **rendered result** of Server Components (payload). The cache is split by route segments which allows invalidation at any level and ensures consistency across concurrent renders.
+- **`<Link>` component**: Routes are automatically prefetched as they become visible in the user's viewport. Prefetching happens when the page first loads or when it comes into view through scrolling.
+- **`router.prefetch()`**: The `useRouter` hook can be used to prefetched routes programmatically.
 
-As users navigate around the app, the router will store the payload of previously fetched segments **and** [prefetched](#prefetching) segments in the cache.
+The`<Link>`'s prefetching behavior is different for static and dynamic routes:
 
-This means, for certain cases, the router can re-use the cache instead of making a new request to the server. This improves performance by avoiding re-fetching data and re-rendering components unnecessarily.
+- [**Static Routes**](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#static-rendering-default): `prefetch` defaults to `true`. The whole route is prefetched and cached.
+- [**Dynamic Routes**](/docs/app/building-your-application/rendering/static-and-dynamic-rendering#dynamic-rendering): `prefetch` default to `false`. Only the shared layout down until the first `loading.js` file is prefetched and cached for 30s. This reduces the cost of fetching an entire dynamic route, and it means you can show an [instant loading state](/docs/app/building-your-application/routing/loading-ui-and-streaming#instant-loading-states).
 
-### Invalidating the Cache
+> **Good to know**:
+>
+> - Prefetching is not enabled in development, only in production.
 
-[Server Actions](/docs/app/building-your-application/data-fetching/server-actions) can be used to revalidate data on-demand by path ([`revalidatePath`](/docs/app/api-reference/functions/revalidatePath)) or by cache tag ([`revalidateTag`](/docs/app/api-reference/functions/revalidateTag)).
+### 2. Caching
 
-### Prefetching
+Next.js has an **in-memory client-side cache** called the [Router Cache](/docs/app/building-your-application/data-fetching/caching#router-cache). As users navigate around the app, the React Server Payload of [prefetched](#prefetching) route segments and visited routes are stored in the Router Cache.
 
-Prefetching is a way to preload a route in the background before it's visited. The rendered result of prefetched routes is added to the router's client-side cache. This makes navigating to a prefetched route near-instant.
+This means on navigation, the cache is reused as much as possible, instead of making a new request to the server; improving performance by reducing the numbers of requests and data transferred.
```

Comment:
I'm assuming the link below has more information over what the caching behavior is / the timeouts of it. Would be good to highlight that explanation is in that router cache doc as right now it only mentions "Learn how to configure"

Reply:
Good point, I'll update the sentence.